server/src/http-exception.filter.ts

```diff
@@ -0,0 +1,17 @@
+import { ExceptionFilter, Catch, ArgumentsHost, HttpException } from '@nestjs/common';
+import { Request, Response } from 'express';
+
+@Catch(HttpException)
```

Comment:
Note:  here's the docs on nest exception filters & error handling.
https://docs.nestjs.com/exception-filters#exception-filters